static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/centesimus/control.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import type { CurrentABTest, SignInGateVariant } from '../../types';
+import { componentName } from '../../component';
+import {
+    hasUserDismissedGate,
+    isNPageOrHigherPageView,
+    isLoggedIn,
+    isInvalidArticleType,
+    isInvalidSection,
+    isIOS9,
+} from '../../helper';
+
+// pull in the show method from the design folder, which has the html template and and click handlers etc.
+import { designShow } from '../design/patientia'; // using the same design as patientia-variant-1
```

Comment:
even though it's using the same design, i think it might be worth separating into it's own file in case changes are made to either test